services/user.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = "",
```

Comment:
Mark Optional params with `Optional` type hint and set default value to None. Then instead of passing empty values on create use checks to fill out the data that actually exists. This would be preferable to writing empty values to database